bn-apps/memberships-management/membership-service/src/main/kotlin/com/r3/businessnetworks/membership/flows/ConfigUtils.kt

```diff
@@ -13,6 +14,10 @@ object ConfigUtils {
         val propertiesFileName = "membership-service.conf"
         val defaultLocation = (Paths.get("cordapps").resolve("config").resolve(propertiesFileName)).toFile()
         return if (defaultLocation.exists()) ConfigFactory.parseFile(defaultLocation)
-            else ConfigFactory.parseFile(File(ConfigUtils::class.java.classLoader.getResource(propertiesFileName).toURI()))
+            else {
+                val url = this.javaClass.getResource("./$propertiesFileName")
```

Comment:
What exactly didn't work for you? The changes that you proposed would attempt to load the config file from the *current class package* with a fallback to the classpath root. Why would you need to put the property files into the same path where your actual classes are?

Reply:
In short my config was not being picked up from the test classpath, i.e. (src/test/resources/)`com/r3/businessnetworks/membership/flows/membership-service.conf`. Maybe i missed something?

Reply:
You can just put it under `resources` folder, i.e. `src/main/resources`

Reply:
@ivanr3 that only worked for me (i.e. another app) after adding the "/membership-service.conf" fallback.

Reply:
Hm that's sounds off. Because this is how configs are loaded in the tests. Can you provide a code sample which doesn't work for you?

Reply:
@ivanr3 It's in a private repo unfortunately. Can ask for you to get access if you need to, but in short it's just a cordapp using memberships as a compile dependency and an /src/main/resources/membership-service.conf

Reply:
Can you just put together a quick sample that you can share?